ops/model.py

```diff
@@ -60,9 +60,13 @@ def get_relation(self, relation_name, relation_id=None):
         """
         return self.relations._get_unique(relation_name, relation_id)
 
-    def get_binding(self, relation):
-        """Get a network space binding for a Relation object."""
-        return self._bindings.get(relation)
+    def get_binding(self, binding_key):
+        """Get a network space binding.
+
+        binding_key -- a name of a relation or a Relation object. Providing a Relation object will return
+                       a Binding object specific to a relation.
+        """
+        return self._bindings.get(binding_key)
 
 
 class ModelCache:
```

Comment:
should we make ModelCache private (_ModelCache), I don't think it is something we actively want charmers to interact with, right?

Reply:
I think I could have a separate PR for this one.

Reply:
See https://github.com/canonical/operator/pull/170